lib/src/widgets/homeScreenWidgets/card_swiper.dart

```diff
@@ -27,8 +24,8 @@ class CardSwiper extends StatelessWidget {
               ),
               child: new Text(
                 isTextFieldEmpty
-                    ? MovieStrings.ifTrendingMoviesTitle
-                    : MovieStrings.ifSearchingMoviesTitle,
+                    ? MoviesStrings.ifTrendingMoviesTitle
+                    : MoviesStrings.ifSearchingMoviesTitle,
```

Comment:
I think the name of the constants would be better without the "if" at the beginning, only trendingMoviesTitle or searchMovieTitle